### DISCUSSION TITLE
为什么ae导出pag文件预览是一片黑的，没有内容啊

### DISCUSSION THREAD

**Discussion by lvqinsuan** [Tencent/libpag#2549] [Q&A]
<img width="960" alt="AfterFX_THrGCjt6Vv" src="https://github.com/user-attachments/assets/344570d1-7ff8-4f86-94e1-d20c923fb26d">
<img width="960" alt="PAGViewer_CjGc9DHiPC" src="https://github.com/user-attachments/assets/cf03e096-c946-4c20-9cc3-61b6b4a272cb">

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在请描述下复现路径和场景，包括系统版本、AE版本、PAGViewer版本等，如果方便请提供下问题对应的aep、pag文件，用于复现该问题

**Comment by jinwuwu001** [comment]
请问该问题是否还需要跟进，若需要请提供上述信息

**Comment by jinwuwu001** [comment]
用户长期无进一步反馈，暂时关闭，有需要可以再开启
